gv-swagger-spec/swagger.yml

```
@@ -0,0 +1,11215 @@
+      operationId: authorized_computing_accounts_create
+      requestBody:
+        content:
+          application/vnd.api+json:
+            schema:
+              $ref: '#/components/schemas/AuthorizedComputingAccountRequest'
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/AuthorizedComputingAccountRequest'
+          multipart/form-data:
+            schema:
+              $ref: '#/components/schemas/AuthorizedComputingAccountRequest'
+        required: true
+      responses:
+        '201':
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: '#/components/schemas/AuthorizedComputingAccountResponse'
+          description: ''
+      tags:
+      - authorized-computing-accounts
+  /v1/authorized-computing-accounts/{id}/:
+    delete:
+      description: viewset allowing create, retrieve, update, delete
+      operationId: authorized_computing_accounts_destroy
+      parameters:
+      - description: A UUID string identifying this Authorized Computing Account.
+        in: path
+        name: id
+        required: true
+        schema:
+          format: uuid
+          type: string
+      responses:
+        '204':
+          description: No response body
+      tags:
+      - authorized-computing-accounts
+    get:
+      description: viewset allowing create, retrieve, update, delete
+      operationId: authorized_computing_accounts_retrieve
+      parameters:
+      - description: endpoint return only specific fields in the response on a per-type
+          basis by including a fields[TYPE] query parameter.
+        explode: false
+        in: query
+        name: fields[authorized-computing-accounts]
+        schema:
+          items:
+            enum:
+            - id
+            - url
+            - display_name
+            - account_owner
+            - api_base_url
+            - auth_url
+            - authorized_capabilities
+            - authorized_operations
+            - authorized_operation_names
+            - configured_computing_addons
+            - credentials
+            - external_computing_service
+            - initiate_oauth
+            - credentials_available
+            type: string
+          type: array
+      - description: A UUID string identifying this Authorized Computing Account.
+        in: path
+        name: id
+        required: true
+        schema:
+          format: uuid
+          type: string
+      - description: include query parameter to allow the client to customize which
+          related resources should be returned.
+        explode: false
+        in: query
+        name: include
+        schema:
+          items:
+            enum:
+            - account_owner
+            - external_computing_service
+            - configured_computing_addons
+            - authorized_operations
+            type: string
+          type: array
+      responses:
+        '200'
```

One typo and one comment. Not really within the scope of this PR, but this feels a bit strange to me that this is a list view, but we only expect to show one item. Maybe it's not too weird though 🤷  

```suggestion
      description: Get resource reference by resource_uri. Even though this is a
        list method, this endpoint returns only one entity
```